### PR TITLE
release: version 2.1

### PR DESCRIPTION
**Pull request by limwa**

### CODE REVIEW

**Comment by limwa** [comment]
I've added the changelog entry and updated the readme.
Just as a side-note: v2 is not equivalent to v2.0. v2 is used in Actions to refer to v2.x, where x is the latest released minor version. This is useful because we can safely refer to v2 without having to worry about breaking changes, but also without having to worry about updating the action to receive security changes, etc.

I recommend that this repo follows the same conventions from here on out.

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,4 +43,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased]: https://github.com/NIAEFEUP/push-to-niployments/compare/v2...HEAD
```
Comment (Naapperas) [inline CHANGELOG.md]: Missing entry for 2.1

**Review by Naapperas — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (limwa) [reply CHANGELOG.md]: I've just realized Github's editor doesn't show the links in the edit mode, so I didn't even realize there were links... Will change now.

Comment (limwa) [reply CHANGELOG.md]: Wait... They're at the bottom of the file... I didn't know you could do that, my bad :clown_face:

Comment (Naapperas) [reply CHANGELOG.md]: Yeah, I only discovered this when actively using the [Keep A Changelog](https://keepachangelog.com/en/1.1.0/) format, its pretty neat :)

**Review by Naapperas — APPROVED** [review]

**Pull request merged**